docs/compendium/substitute.md

```diff
@@ -25,7 +36,90 @@ introduction...
 ### Error responses
 
 ```json
+401 Unauthorized
+
+{"error":"not authenticated or not allowed"}
+```
+
+```json
+404 Not Found
+
+{"error":"compendia ids not found"}
```

Comment:
You have this below in the `POST` request. Very good. Suggest to remove the section on the `GET` request until you provide a use case for it.